seq_logger.js

```diff
@@ -148,12 +139,19 @@ class SeqLogger {
     }
 
     _toWireFormat(event) {
+        const level = typeof event.level === 'string' ? event.level : undefined;
+        const timestamp = event.timestamp instanceof Date ? event.timestamp : new Date();
```

Comment:
Do we want this to be the case even if the timestamp is some other defined value, like an ISO8601 date string?

Reply:
Thought about this - it's a bit tricky to check that a string is valid ISO-8601 - could add this in future 👍